tests/cli/test_cmds_local_run.py

```diff
@@ -2081,6 +2081,9 @@ def test_run_docker_container_assume_aws_role(
     assert 0 == return_code
 
 
+fake_service_names = ["fake_service", "override_service_name"]
```

Comment:
i'd suggest inlining this just to be consistent with how we generally parametrize :)

Reply:
I can either 
a. parametrize inline and hardcode `service="fake_service"` within the test, 
b. add `service="fake_service"` to the start of the file and replace fake_service everywhere
c. keep it as-is

what's the best in your opinion?

Reply:
maybe something like:
```python
@mark.parametrize("original_service_name, override_service_name", (
    ("fake_service", "fake_service"), # no service override
    ("fake_service", "super_fake_service"), # service override
))
...
mock_service_manifest.get_service = mock.MagicMock(
        return_value=override_service_name
    )
...
    return_code = run_docker_container(
        docker_client=mock_docker_client,
        service=original_service_name,
        ...
    )
...
assert decrypt_kwargs["service_name"] == override_service_name
```
would work?

Reply:
Thanks, didn't think of having both of em in the parameters :doh: